pysal/contrib/geotable/utils.py

```diff
@@ -50,13 +49,40 @@ def to_gdf(df, geom_col='geometry', **kw):
     """
     from geopandas import GeoDataFrame
     from shapely.geometry import asShape as sShape
-    df[geom_col] = df[geom_col].apply(sShape)
-    return GeoDataFrame(df, geometry=geom_col, **kw)
+    out = df.copy()
+    out[geom_col] = out[geom_col].apply(sShape)
+    out = GeoDataFrame(out, geometry=geom_col, **kw)
+    return out
 
-def insert_metadata(df, obj, name=None, inplace=True, overwrite=False):
+def insert_metadata(df, obj, name=None, inplace=False, overwrite=False):
```

Comment:
Sorry for my ignorance, what is this one for? Is this allowing the user to do `to_df` and `to_gdf` inplace? I don't think so and I'm sure I'm missing something.